dashboard/test/ui/features/applab/eyes2.feature

```diff
@@ -35,6 +35,13 @@ Scenario: Applab embedded level
   And I see no difference for "embedded level"
   Then I close my eyes
 
+Scenario: Applab widget mode
+  When I open my eyes to test "Applab widget mode"
+  And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/22"
+  And I wait until element "#reset" is visible
```

Comment:
I thought widget mode didn't show the reset button?

Reply:
Ahh sorry, this is a button within the widget app with id `reset`. The reset button has id `resetButton`. I'll see what I can do to edit the widget app a bit to make this more clear.

Reply:
Fixed!